cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/BuilderUtils.java

```diff
@@ -89,14 +90,14 @@ public static void compileWithTestsAndWrite(Path sourceRootPath, boolean offline
                 outStream.println();
                 compiler.write(packages);
             } else {
-                outStream.println("No ballerina source files found to compile");
+                throw new BLangCompilerException("no ballerina source files found to compile");
```

Comment:
Are we walking into the folder to check any bal files exists ?

Reply:
The check was moved to the Compiler